unit15/Ex1/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://kit-pro.fontawesome.com/releases/v5.12.0/css/pro.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="content">
+        <div class="formAdd">
+        <form id="form" style="display: none;"> 
+            <div class="form-group">
+                <label for="name">Name</label>
+                <input type="text" class="form-control" id="name"/>
+            </div>
+            <div class="form-group">
+                <label for="">Gender</label>
+                <select class="form-control" id="gender">
+                    <option>Nam</option>
+                    <option>Nữ</option>
+                </select>
+            </div>
+            <div class="form-group">
+                <label for="name">Age</label>
+                <input type="number" class="form-control" id="age">
+            </div>
+            <button type="button" class="btn btn-primary" id="add_form" onclick="addStudent()">Add form</button>
+        </form>
+        <div class="button">
```

Comment:
```suggestion
        </form>

        <div class="button">
```

Các block khác nhau thì viết cách nhau ra